engine/runtime/src/main/java/org/enso/interpreter/Language.java

```diff
@@ -108,12 +110,15 @@ protected Context createContext(Env env) {
     Context context =
         new Context(
             this, getLanguageHome(), env, notificationHandler, lockManager, distributionManager);
-    InstrumentInfo idValueListenerInstrument =
-        env.getInstruments().get(IdExecutionInstrument.INSTRUMENT_ID);
-    idExecutionInstrument = env.lookup(idValueListenerInstrument, IdExecutionInstrument.class);
+    idExecutionInstrument =
+        Optional.ofNullable(env.getInstruments().get(IdExecutionService.INSTRUMENT_ID))
+            .map(
+                idValueListenerInstrument ->
+                    env.lookup(idValueListenerInstrument, IdExecutionService.class));
```

Comment:
Why do we allow `null` here? Shouldn't we throw an exception if the instrument is not available?

Reply:
Is it ever possible to not have the instrument loaded in any other scenario other than a sever misconfiguration?

Reply:
That would make it impossible to setup unit tests in `runtime`. All tests that depend on the instrumentation have been moved to `runtime-with-instruments/src/test`

Reply:
Well the instruments should be already available in `runtime-with-instruments`, so I'm not sure if I understand the issue here. Is there a scenario when we run the runtime and the instruments are unavailable? In the tests it sounds like they totally should be available.

Reply:
Yes, all tests that are in `runtime/src/test`